ansible/roles/kubernetes/tasks/override_facts.yml

```diff
@@ -2,6 +2,7 @@
 - name: Update {{ kube_script_dir }} if this is atomic
   set_fact:
     kube_script_dir: "/usr/local/libexec/kubernetes"
+    etcd_script_dir: "/usr/local/libexec/etcd"
```

Comment:
This isn't working, because the the etcd task is executing before the kubernetes task does.